src/libraries/Microsoft.Extensions.Diagnostics.Abstractions/src/Metrics/MetricsBuilderExtensions.Rules.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics.Metrics;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Diagnostics.Metrics
+{
+    /// <summary>
+    /// Extension methods for <see cref="IMetricsBuilder"/> to enable or disable metrics.
```

Comment:
>Extension methods for <see cref="IMetricsBuilder"/> to enable or disable metrics. [](http://example.com/codeflow?start=8&length=81)

This is a different description than what is mentioned in the `.Listeners.cs` file. We have the tool that port the source doc comments to the public docs so I am not sure if this can cause issues. 

CC @carlossanlop